diff --git a/docs/website/src/theme/Footer/Links/MultiColumn/index.tsx b/docs/website/src/theme/Footer/Links/MultiColumn/index.tsx
index f06d1ea4b..dec1797ad 100644
--- a/docs/website/src/theme/Footer/Links/MultiColumn/index.tsx
+++ b/docs/website/src/theme/Footer/Links/MultiColumn/index.tsx
@@ -1,6 +1,5 @@
 import React, { type ReactNode } from "react";
 import clsx from "clsx";
-import { ThemeClassNames } from "@docusaurus/theme-common";
 import LinkItem from "@theme/Footer/LinkItem";
 import type { Props } from "@theme/Footer/Links/MultiColumn";

@@ -24,15 +23,9 @@ function ColumnLinkItem({ item }: { item: ColumnItemType }) {

 function Column({ column }: { column: ColumnType }) {
   return (
-    <div
-      className={clsx(
-        ThemeClassNames.layout.footer.column,
-        "col footer__col",
-        column.className,
-      )}
-    >
-      <div className="footer__title">{column.title}</div>
-      <ul className="footer__items clean-list">
+    <div className="flex flex-col gap-6">
+      <div className="font-bold min-h-5">{column.title}</div>
+      <ul className="flex flex-col gap-6">
         {column.items.map((item, i) => (
           <ColumnLinkItem key={i} item={item} />
         ))}
@@ -43,7 +36,7 @@ function Column({ column }: { column: ColumnType }) {

 export default function FooterLinksMultiColumn({ columns }: Props): ReactNode {
   return (
-    <div className="row footer__links">
+    <div className="laptop:flex laptop:flex-row tablet:grid tablet:grid-cols-2 laptop:gap-[6.25rem] gap-y-7 flex flex-col text-sm pb-12 tablet:pb-[3.625rem]">
       {columns.map((column, i) => (
         <Column key={i} column={column} />
       ))}
